### PR TITLE
Twig pay icon

### PR DESCRIPTION
**Pull request by taulanti**
** Checklist **
- [x] All icons have a corresponding entry in `db/payment_icons.yml`
- [x] I have followed the icon guidelines detailed in the [CONTRIBUTING.md](https://github.com/activemerchant/payment_icons/blob/master/CONTRIBUTING.md) file
- [x] I have optimized the icon with [SVGO](https://jakearchibald.github.io/svgomg/)
- [x] I am confident that all icons are clear and easy to read/understand
- [ ] I have provided a link to the brand icon’s brand guidelines whenever possible.
- [x] I recognize that if my icon is not approved by the Shopify Partners team it may not receive review nor merger.

If this pull request is not adding new icons, you can remove this checklist.

### CODE REVIEW

**Comment by taulanti** [comment]
Hi @larouxn, is there a queue to do the reviews or it's done by asking explicitly?

**Comment by taulanti** [comment]
HI @larouxn thanks for taking this in consideration. Added the prefix pi-

**Review by larouxn — CHANGES_REQUESTED** [review]
👋 Looks like the border might be inverted or something. Can we take a look at the [template](https://github.com/activemerchant/payment_icons/blob/2dcdc623b7ea86cfe73469eeb52c97d7c8bb2c24/CONTRIBUTING.md#template) to see how it should be implemented? Thank you 🙇‍♂️ 
Screen Shot 2022-08-17 at 11 03 38

**Review by larouxn — APPROVED** [review]
Looks good to me 🎩
Screen Shot 2022-08-17 at 20 17 52

Just need to rebase against latest `master`. Alternatively, manually move the `payment_icons.yml` file entry to somewhere towards the middle of the file.

File: app/assets/images/payment_icons/twig_pay.svg [hunk 1]
```diff
@@ -0,0 +1 @@
....203v.52c0 .11-.088.203-.213.203h-.468c.187.286.281.588.281.905.026 1.097-.858 1.955-1.982 1.935a2 2 0 01-.816-.172.375.375 0 00-.224.359c0 .192.219.327.53.327h1.353c1.05 0 1.716.52 1.716 1.368zm-3.443-3.828c0 .5.39.9.895.9.52 0 .926-.39.926-.894a.914.914 0 00-.92-.926c-.5 0-.9.406-.9.92zm2.434 3.917c0-.328-.228-.453-.707-.453h-1.175c-.479 0-.708.125-.708.453 0 .37.494.578 1.295.582.801-.004 1.295-.212 1.295-.582zm-6.23-5.633a.203.203 0 01-.016.068L13.8 14.157c-.026.068-.12.125-.203.125h-.613c-.084 0-.177-.057-.203-.125l-1.243-2.979-1.244 2.98c-.026.067-.12.124-.203.124h-.613c-.083 0-.177-.057-.203-.125L7.48 9.945a.203.203 0 01-.016-.068c0-.067.052-.11.12-.11h.91a.2.2 0 01.192.13l1.098 3.1 1.128-3.1a.199.199 0 01.193-.13h.863a.2.2 0 01.193.13l1.128 3.1 1.098-3.1a.199.199 0 01.192-.13h.91c.068 0 .12.043.12.11zm-7.791 3.879c-.357.394-.882.62-1.47.62-1.086 0-1.954-.77-1.954-1.94V10.85H3.59a.14.14 0 01-.14-.14v-.797a.14.14 0 01.14-.14h.8V8.068a.14.14 0 01.14-.14h.853a.14.14 0 01.14.14v1.705h.613c.084 0 .16.05.192.127l.351.824c.025.058-.015.125-.077.126H5.524v1.587c0 .494.366.819.824.819.433 0 .686-.276.854-.508a.207.207 0 01.357.043l.301.706a.244.244 0 01-.041.259z" fill="#fff"/></svg>
```
Comment (tauthomas01) [inline app/assets/images/payment_icons/twig_pay.svg:1]: Could we remove the unused `id` and other svg attributes? Those aren't needed.

```suggestion
<svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 38 24" width="38" height="24" aria-labelledby="pi-twig_pay"><title id="pi-twig_pay">Twig Pay</title><path d="M35 0H3C1.3 0 0 1.3 0 3v18c0 1.7 1.4 3 3 3h32c1.7 0 3-1.3 3-3V3c0-1.7-1.4-3-3-3z" opacity=".07"/><linearGradient id="SVGID_1_" gradientUnits="userSpaceOnUse" x1="1" y1="12" x2="37" y2="12"><stop offset="0" stop-color="#00dab9"/><stop offset=".998" stop-color="#72b0da"/></linearGradient><path d="M35 1c1.1 0 2 .9 2 2v18c0 1.1-.9 2-2 2H3c-1.1 0-2-.9-2-2V3c0-1.1.9-2 2-2h32" fill="url(#SVGID_1_)"/><path d="M27.624 11.124a.484.484 0 01.146.365c0 .148-.048.27-.146.365a.52.52 0 01-.375.142h-.513V10.98h.513c.153 0 .277.048.375.144zm1.683 1.372h.934l-.468-1.283-.466 1.283zm5.243-2.4v3.858a.329.329 0 01-.329.328h-8.656a.329.329 0 01-.328-.328v-3.858c0-.181.147-.328.328-.328h8.656c.182 0 .329.147.329.328zm-6.456 2.213c.217-.208.327-.48.327-.815 0-.34-.11-.615-.327-.823-.217-.207-.497-.312-.845-.312h-1.066c-.057 0-.098.04-.098.102v3.134c0 .068.032.102.098.102h.45c.068 0 .103-.034.103-.102v-.978h.513c.348 0 .628-.103.845-.308zm3.245 1.274l-1.253-3.133c-.023-.062-.064-.094-.124-.094h-.372c-.06 0-.1.032-.125.094l-1.256 3.133c-.026.07.002.114.075.114h.498a.132.132 0 00.123-.091l.201-.555h1.336l.203.557a.128.128 0 00.119.092h.5c.073 0 .103-.046.075-.117zm2.499-3.224h-.528c-.052 0-.096.03-.125.086l-.695 1.272-.696-1.27c-.03-.059-.07-.088-.121-.088h-.532c-.076 0-.105.043-.069.118l1.092 1.907v1.21c0 .069.034.103.1.103h.455c.066 0 .098-.034.098-.102v-1.21l1.092-1.908c.03-.075.004-.118-.071-.118zM16.8 7.017a.712.712 0 00-.702.697.71.71 0 00.702.691c.38 0 .697-.317.697-.691a.704.704 0 00-.697-.697zm.562 2.891v4.234c0 .088-.057.14-.146.14h-.842c-.094 0-.14-.047-.14-.14V9.908c0-.093.046-.14.14-.14h.842c.089 0 .146.052.146.14zm5.487 5.513c0 1.038-.919 1.664-2.304 1.68-1.385-.016-2.304-.642-2.304-1.68 0-.333.104-.616.293-.836a1.145 1.145 0 01-.293-.786c0-.422.203-.79.536-.973a1.918 1.918 0 01-.421-1.227c-.021-1.124.837-1.987 1.945-1.972.182 0 .489.058.76.146h1.398c.13 0 .213.073.213.203v.52c0 .11-.088.203-.213.203h-.468c.187.286.281.588.281.905.026 1.097-.858 1.955-1.982 1.935a2 2 0 01-.816-.172.375.375 0 00-.224.359c0 .192.219.327.53.327h1.353c1.05 0 1.716.52 1.716 1.368zm-3.443-3.828c0 .5.39.9.895.9.52 0 .926-.39.926-.894a.914.914 0 00-.92-.926c-.5 0-.9.406-.9.92zm2.434 3.917c0-.328-.228-.453-.707-.453h-1.175c-.479 0-.708.125-.708.453 0 .37.494.578 1.295.582.801-.004 1.295-.212 1.295-.582zm-6.23-5.633a.203.203 0 01-.016.068L13.8 14.157c-.026.068-.12.125-.203.125h-.613c-.084 0-.177-.057-.203-.125l-1.243-2.979-1.244 2.98c-.026.067-.12.124-.203.124h-.613c-.083 0-.177-.057-.203-.125L7.48 9.945a.203.203 0 01-.016-.068c0-.067.052-.11.12-.11h.91a.2.2 0 01.192.13l1.098 3.1 1.128-3.1a.199.199 0 01.193-.13h.863a.2.2 0 01.193.13l1.128 3.1 1.098-3.1a.199.199 0 01.192-.13h.91c.068 0 .12.043.12.11zm-7.791 3.879c-.357.394-.882.62-1.47.62-1.086 0-1.954-.77-1.954-1.94V10.85H3.59a.14.14 0 01-.14-.14v-.797a.14.14 0 01.14-.14h.8V8.068a.14.14 0 01.14-.14h.853a.14.14 0 01.14.14v1.705h.613c.084 0 .16.05.192.127l.351.824c.025.058-.015.125-.077.126H5.524v1.587c0 .494.366.819.824.819.433 0 .686-.276.854-.508a.207.207 0 01.357.043l.301.706a.244.244 0 01-.041.259z" fill="#fff"/></svg>
```

**Review by tauthomas01 — CHANGES_REQUESTED** [review]
Looks good. I left a comment to clean up the svg code.

**Comment by taulanti** [comment]
Hello @tauthomas01, removed Ids also merged with the latest version of master to remove conflicts.

File: app/assets/images/payment_icons/twig_pay.svg [hunk 2]
```diff
@@ -0,0 +1 @@
...
```
Comment (tauthomas01) [inline app/assets/images/payment_icons/twig_pay.svg:1]: We'll need to add `role="img"` in the `<svg>` as tests are failing

```suggestion
<svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 38 24" width="38" height="24" aria-labelledby="pi-twig_pay" role="img"><title id="pi-twig_pay">Twig Pay</title><path d="M35 0H3C1.3 0 0 1.3 0 3v18c0 1.7 1.4 3 3 3h32c1.7 0 3-1.3 3-3V3c0-1.7-1.4-3-3-3z" opacity=".07"/><linearGradient id="pi-twig_pay-SVGID_1_" gradientUnits="userSpaceOnUse" x1="1" y1="12" x2="37" y2="12"><stop offset="0" stop-color="#00dab9"/><stop offset=".998" stop-color="#72b0da"/></linearGradient><path d="M35 1c1.1 0 2 .9 2 2v18c0 1.1-.9 2-2 2H3c-1.1 0-2-.9-2-2V3c0-1.1.9-2 2-2h32" fill="url(#pi-twig_pay-SVGID_1_)"/><path d="M27.624 11.124a.484.484 0 01.146.365c0 .148-.048.27-.146.365a.52.52 0 01-.375.142h-.513V10.98h.513c.153 0 .277.048.375.144zm1.683 1.372h.934l-.468-1.283-.466 1.283zm5.243-2.4v3.858a.329.329 0 01-.329.328h-8.656a.329.329 0 01-.328-.328v-3.858c0-.181.147-.328.328-.328h8.656c.182 0 .329.147.329.328zm-6.456 2.213c.217-.208.327-.48.327-.815 0-.34-.11-.615-.327-.823-.217-.207-.497-.312-.845-.312h-1.066c-.057 0-.098.04-.098.102v3.134c0 .068.032.102.098.102h.45c.068 0 .103-.034.103-.102v-.978h.513c.348 0 .628-.103.845-.308zm3.245 1.274l-1.253-3.133c-.023-.062-.064-.094-.124-.094h-.372c-.06 0-.1.032-.125.094l-1.256 3.133c-.026.07.002.114.075.114h.498a.132.132 0 00.123-.091l.201-.555h1.336l.203.557a.128.128 0 00.119.092h.5c.073 0 .103-.046.075-.117zm2.499-3.224h-.528c-.052 0-.096.03-.125.086l-.695 1.272-.696-1.27c-.03-.059-.07-.088-.121-.088h-.532c-.076 0-.105.043-.069.118l1.092 1.907v1.21c0 .069.034.103.1.103h.455c.066 0 .098-.034.098-.102v-1.21l1.092-1.908c.03-.075.004-.118-.071-.118zM16.8 7.017a.712.712 0 00-.702.697.71.71 0 00.702.691c.38 0 .697-.317.697-.691a.704.704 0 00-.697-.697zm.562 2.891v4.234c0 .088-.057.14-.146.14h-.842c-.094 0-.14-.047-.14-.14V9.908c0-.093.046-.14.14-.14h.842c.089 0 .146.052.146.14zm5.487 5.513c0 1.038-.919 1.664-2.304 1.68-1.385-.016-2.304-.642-2.304-1.68 0-.333.104-.616.293-.836a1.145 1.145 0 01-.293-.786c0-.422.203-.79.536-.973a1.918 1.918 0 01-.421-1.227c-.021-1.124.837-1.987 1.945-1.972.182 0 .489.058.76.146h1.398c.13 0 .213.073.213.203v.52c0 .11-.088.203-.213.203h-.468c.187.286.281.588.281.905.026 1.097-.858 1.955-1.982 1.935a2 2 0 01-.816-.172.375.375 0 00-.224.359c0 .192.219.327.53.327h1.353c1.05 0 1.716.52 1.716 1.368zm-3.443-3.828c0 .5.39.9.895.9.52 0 .926-.39.926-.894a.914.914 0 00-.92-.926c-.5 0-.9.406-.9.92zm2.434 3.917c0-.328-.228-.453-.707-.453h-1.175c-.479 0-.708.125-.708.453 0 .37.494.578 1.295.582.801-.004 1.295-.212 1.295-.582zm-6.23-5.633a.203.203 0 01-.016.068L13.8 14.157c-.026.068-.12.125-.203.125h-.613c-.084 0-.177-.057-.203-.125l-1.243-2.979-1.244 2.98c-.026.067-.12.124-.203.124h-.613c-.083 0-.177-.057-.203-.125L7.48 9.945a.203.203 0 01-.016-.068c0-.067.052-.11.12-.11h.91a.2.2 0 01.192.13l1.098 3.1 1.128-3.1a.199.199 0 01.193-.13h.863a.2.2 0 01.193.13l1.128 3.1 1.098-3.1a.199.199 0 01.192-.13h.91c.068 0 .12.043.12.11zm-7.791 3.879c-.357.394-.882.62-1.47.62-1.086 0-1.954-.77-1.954-1.94V10.85H3.59a.14.14 0 01-.14-.14v-.797a.14.14 0 01.14-.14h.8V8.068a.14.14 0 01.14-.14h.853a.14.14 0 01.14.14v1.705h.613c.084 0 .16.05.192.127l.351.824c.025.058-.015.125-.077.126H5.524v1.587c0 .494.366.819.824.819.433 0 .686-.276.854-.508a.207.207 0 01.357.043l.301.706a.244.244 0 01-.041.259z" fill="#fff"/></svg>
```

**Review by tauthomas01 — CHANGES_REQUESTED** [review]

**Comment by taulanti** [comment]
@tauthomas01 approved the changes

**Review by tauthomas01 — APPROVED** [review]

**Comment by taulanti** [comment]
@tauthomas01 Hey, hope you're doing good. The PR has been approved 12 days ago, when a new release is expected? thanks.

**Comment by larouxn** [comment]
Hey all, Thomas and I no longer maintain this gem but glad to merge. I would expect this to be live within the month or so. 🤞

**Pull request merged**